src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -215,5 +215,17 @@ public override Task SendUsersAsync(IReadOnlyList<string> userIds, string method
                 return userIds.Contains(connection.UserIdentifier);
             });
         }
+
+        private Task SafeWriteAsync(HubConnectionContext connection, InvocationMessage message)
+        {
+            try
+            {
+                return connection.WriteAsync(message);
```

Comment:
What if we `await`ed here (to get all the exceptions), logged the exception and ignored it? Is there actually any value in sending this exception back to the client? What are they going to do about it?

Plus, things are worse in the Redis case. If **your** server fails to send the message you get an exception but if a **different** one fails to send to one of it's clients you get nothing.